### ISSUE TITLE
https://localhost/articles - "This site can’t be reached"

### ISSUE THREAD

**Issue by darius-v** [ger86/symfony-docker#20]
At first it worked but after something it stopped working. The error is totally unclear, like its just not working server. But when I go to localhost - it loads homepage, so it means server works. How can this be? 

And I do not understand where to check for logs. I just see symbolyc links to symbolic links

![image](https://github.com/ger86/symfony-docker/assets/6602631/26466e4d-24c5-46fb-b23d-88a143c547b0)

**Comment by ger86** [comment]
Hi Darius,

You have to go inside the container. In VSCode, you can use the extension Remote Containers. Maybe in PHPStorm is something similar. Then, you should see the `var` and `vendor` folders

**Comment by darius-v** [comment]
oh but they should not be  visible in linux file system still by default, only through Remote containers IDE feature?

**Comment by darius-v** [comment]
btw I fixed /articles problem by replacing nginx config from another person and removing evrything related to https.
ANd then I wnet to /articles url - same problem - still it was not loading. but / was loading. THen I decided to check network setting and after some plays it started loading /articles. Maybe there was browser cache problem. /articles was trying to go to https while I went to http on localhost.

Maybe same problem was with this image config, maybe I will check if will not forget next day.
